### PR TITLE
Refactor Selenium tests to take screenshot when password fails

### PR DESCRIPTION
**Pull request by bgavrilMS**

### CODE REVIEW

File: tests/Microsoft.Identity.Test.Integration.netfx/Infrastructure/SeleniumLoginDriver.cs [hunk 1]
```diff
@@ -0,0 +1,194 @@
+﻿using System;
```
Comment (jennyf19) [inline tests/Microsoft.Identity.Test.Integration.netfx/Infrastructure/SeleniumLoginDriver.cs:1]:
```suggestion
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

﻿using System;
```

File: tests/Microsoft.Identity.Test.Integration.netfx/Infrastructure/SeleniumLoginDriver.cs [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+            LabUser user,
+            TestContext testContext,
+            CancellationToken cancellationToken)
+        {
+            _driver = driver;
+            _user = user;
+            _testContext = testContext;
+            _cancellationToken = cancellationToken;
+            _htmlFieldIds = new UserInformationFieldIds(user);
+
```
Comment (jennyf19) [inline tests/Microsoft.Identity.Test.Integration.netfx/Infrastructure/SeleniumLoginDriver.cs:34]:
```suggestion
```

**Review by jennyf19 — APPROVED** [review]
nice updates. :shipit:

**Comment by bgavrilMS** [comment]
There are still problems with this, will look at it locally

**Pull request closed without merging**